helix-rest/src/test/java/org/apache/helix/rest/server/AbstractTestClass.java

```diff
@@ -316,7 +316,7 @@ private void setupZooKeepers() {
   }
 
   protected void setupHelixResources() {
-    _clusters = createClusters(3);
+    _clusters = createClusters(4);
```

Comment:
Just curious, why we need to modify the cluster count when adding new tests? Can't we create the cluster inside the new test cases? Or are we supposed to modify more than one test file every time when we need to add a test case?

Reply:
I agree that the optimal way is for each test to create cluster inside the test itself. However, this need a larger code refactor for all our helix-rest integration test.